dev-tools/validation/test_adr_review_status.py

```
@@ -0,0 +1,577 @@
+import unittest
+from pathlib import Path
+
+
+ROOT = Path(__file__).resolve().parents[2]
+SCRIPT = ROOT / "dev-tools/validation/check-adr-review-status.py"
+
+
+def load_validator():
+    spec = importlib.util.spec_from_file_location("adr_review_status_validator", SCRIPT)
+    if spec is None or spec.loader is None:
+        raise AssertionError("could not load ADR review status validator")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)